src/app/config.module.ts

```diff
@@ -32,6 +32,14 @@ import { FeatureResolverModule } from './shared/feature-resolver/feature-resolve
         colors: ['#EEC200', '#F7902D', '#E8654B', '#C44660', '#923768', '#5B2F60', '#27244A', '#001429']
       }
     },
+    {
+      provide: ALTERNATE_COLOR_PALETTES,
+      multi: true,
+      useValue: {
+        key: GRAY_COLOR_PALETTE,
+        colors: ['#3F474A', '#565F62', '#6C767A', '#828D91', '#97A3A7', '#ACB9BD', '#C1CFD4', '#D6E5EA']
```

Comment:
`#3F474A`, `#565F62`, `#6C767A`, `#828D91`, `#97A3A7`, `#ACB9BD`, `#C1CFD4`, `#D6E5EA` For everyone's viewing pleasure

Reply:
hah 🤣